rs/bitcoin/ckbtc/minter/src/lib.rs

```diff
@@ -166,19 +166,14 @@ async fn fetch_main_utxos(main_account: &Account, main_address: &BitcoinAddress)
 /// Returns the minimum withdrawal amount based on the current median fee rate (in millisatoshi per byte).
 /// The returned amount is in satoshi.
 fn compute_min_withdrawal_amount(
-    btc_network: Network,
     median_fee_rate_e3s: MillisatoshiPerByte,
+    min_withdrawal_amount: u64,
 ) -> u64 {
     const PER_REQUEST_RBF_BOUND: u64 = 22_100;
     const PER_REQUEST_VSIZE_BOUND: u64 = 221;
     const PER_REQUEST_MINTER_FEE_BOUND: u64 = 305;
     const PER_REQUEST_KYT_FEE: u64 = 2_000;
 
-    let min_withdrawal_amount = match btc_network {
-        Network::Testnet | Network::Regtest => 10_000,
-        Network::Mainnet => 100_000,
-    };
-
     let median_fee_rate = median_fee_rate_e3s / 1_000;
     ((PER_REQUEST_RBF_BOUND
         + PER_REQUEST_VSIZE_BOUND * median_fee_rate
```

Comment:
The current logic increments the fee in chunks of 50k satoshis. Do we want to keep it that way?
Before, it was 100k -> 150k -> 200k... and now it would be 50k -> 100k -> 150k.

I think we can leave it like that. What do you think?

Reply:
Agree that we can leave it like that.